tests/batch_test.go

```diff
@@ -1,106 +1,129 @@
 package easypost_test
 
 import (
-	"time"
+	"reflect"
+	"strings"
 
 	"github.com/EasyPost/easypost-go/v2"
 )
 
-func (c *ClientTests) TestBatchCreateAndBuy() {
+func (c *ClientTests) TestBatchCreate() {
 	client := c.TestClient()
-	assert, require := c.Assert(), c.Require()
-	// We create Address and Parcel objects. We then try to create a Batch
-	// containing a shipment. Finally, we assert on saved and returned data.
-	from, err := client.CreateAddress(
-		&easypost.Address{
-			Company: "EasyPost",
-			Street1: "One Montgomery St",
-			Street2: "Ste 400",
-			City:    "San Francisco",
-			State:   "CA",
-			Zip:     "94104",
-			Phone:   "415-555-1212",
-		},
+	assert := c.Assert()
+
+	batch, _ := client.CreateBatch(
+		c.fixture.BasicShipment(),
 		nil,
 	)
-	require.NoError(err)
 
-	parcel, err := client.CreateParcel(
-		&easypost.Parcel{PredefinedPackage: "RegionalRateBoxA", Weight: 64},
+	assert.Equal(reflect.TypeOf(&easypost.Batch{}), reflect.TypeOf(batch))
+	assert.True(strings.HasPrefix(batch.ID, "batch_"))
+	assert.NotNil(batch.Shipments)
+}
+
+func (c *ClientTests) TestBatchRetrieve() {
+	client := c.TestClient()
+	assert := c.Assert()
+
+	batch, _ := client.CreateBatch(
+		c.fixture.BasicShipment(),
+		nil,
 	)
-	require.NoError(err)
-
-	shipments := []*easypost.Shipment{
-		{
-			ToAddress: &easypost.Address{
-				Name:    "Bugs Bunny",
-				Street1: "4000 Warner Blvd",
-				City:    "Burbank",
-				State:   "CA",
-				Zip:     "91522",
-				Phone:   "818-555-1212",
-			},
-			FromAddress: from,
-			Parcel:      parcel,
-			Reference:   "1234567890",
-			Carrier:     "USPS",
-			Service:     "Priority",
+
+	retrievedBatch, _ := client.GetBatch(batch.ID)
+
+	assert.Equal(reflect.TypeOf(&easypost.Batch{}), reflect.TypeOf(retrievedBatch))
+	assert.Equal(batch.ID, retrievedBatch.ID)
+}
+
+/* This test is currently skipped because we need to fix the typo of ListBatchesResult from `insurance` to `batch` in next PR
```

Comment:
Let's not have large amounts of commented out code, instead, let's skip the test instead. The native Go `testing` package has this available to do just that: 

```go
t.Skip("This test is currently skipped because we need to fix the typo of ListBatchesResult from `insurance` to `batch` in next PR")
```

You'll probably need to pass in the testing package like the ClientTests is above.